lib/sun_times.ex

```diff
@@ -79,11 +79,13 @@ defmodule SunTimes do
 
         # T = H + RA - (0.06571 * t) - 6.622
         local_mean_time = suns_local_hour_hours + sun_right_ascension_hours - (0.06571 * approximate_time) - 6.622
+        local_mean_time = if event == :set, do: local_mean_time + 24, else: local_mean_time
```

Comment:
Can you add a comment that sunset ends up with a negative local time, and it needs to be a positive number of hours?